browser/js/components/Comment/test.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+import React, {
```

Comment:
Okay I am thinking thoughts because we are reusing so many of the same variables and there are so many! I'll let you know as my thought progresses.